docs/_includes/js/scrollspy.html

```diff
@@ -65,7 +65,7 @@ <h2 id="scrollspy-usage">Usage</h2>
   </div>
 
   <h3>Requires relative positioning</h3>
-  <p>No matter the implementation method, scrollspy requires the use of <code>position: relative;</code> on the element you're spying on. In most cases this is the <code>&lt;body&gt;</code>.</p>
+  <p>No matter the implementation method, scrollspy requires the use of <code>position: relative;</code> on the element you're spying on. In most cases this is the <code>&lt;body&gt;</code>. In some cases though, you may use another element, this element will require a valid height and overflow to be set on the element being spied on.</p>
```

Comment:
Adjusted and simplified the grammar of the sentance. I think it should be a bit clearer now. Let me know though if you have anymore feedback :+1: